.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I think we want there to be some definitions here (and then reapply) -- here's what I've put in place on the Facet branch that [swaps FAST for Stencil](https://github.com/pulumi/facet/pull/16) (which I'd love to be able to merge once https://github.com/pulumi/theme/pull/45 is done), which has a few things we've already standardized on (like double-quotes, 4-space tabs and the like):

https://github.com/pulumi/facet/blob/cnunciato/swap-fast-for-stencil/.prettierrc.json